wp-content/themes/core/pcss/content/_all.pcss

```diff
@@ -30,8 +30,7 @@
 @import "single/post";
 
 /* Panels */
-@import "panel/default";
```

Comment:
Let's reverse this, in porting this down we want to make sure it's consistent with how we handle the other partials on the content directory. Nix the all partial and import them as part of the all for the content directory.